src/include/daos_obj_class.h

```diff
@@ -11,7 +11,60 @@ extern "C" {
 #endif
 
 /**
- * Predefined object classes
+ * Define the object data redundancy method. Encoded over 8 bits in the object ID.
+ * The number of redundancy groups is encoded separately in the object metadata.
+ */
+enum daos_obj_redun {
+	/**
+	 * No data protection, aka "single replica".
+	 */
+	OR_RP_1		= 1,
+
+	/**
+	 * Static N-way replicated object (OC_RP_N).
+	 * The number of redundancy group is hardcoded in the object metadata.
+	 */
+	OR_RP_2		= 8,
```

Comment:
why do you skip 2-7

Reply:
I am not sure, @johannlombardi made this, i think i could fix it.